src/scripts/main.js

```diff
@@ -2,10 +2,20 @@
 
 function waitFor(element, eventName) {
   // write your code here
+  return new Promise((resolve) => {
+    element.addEventListener(eventName, () => {
+      // eslint-disable-next-line
+      resolve(`It was ${eventName} on the element: ${element.nodeName}, id: ${element.id}.`);
```

Comment:
If your line length greater than 80 chars split it
```suggestion
      resolve(
        `It was ${eventName} on the element: ${element.nodeName}, id: ${element.id}.`
      );
```